### PR TITLE
Cleanup without database...

### PR DESCRIPTION
**Pull request by jacklul**
Prevent running /cleanup when there is no DB connection...

_I don't think it will clean up the universe in that scenario._

### CODE REVIEW

File: src/Commands/AdminCommands/CleanupCommand.php [hunk 1]
```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```
Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Thought it makes sense to keep this private, as not to expose any info in a group.
Mind explaining why the change to `getChat` now?

File: src/Commands/AdminCommands/CleanupCommand.php [hunk 2]
```diff
@@ -319,6 +324,25 @@ private function getQueries($settings)
...
+     * @return \Longman\TelegramBot\Entities\ServerResponse
+     */
+    public function executeNoDb()
+    {
+        $message = $this->getMessage();
+        $chat_id = $message->getChat()->getId();
+
+        $data = [
+            'chat_id'    => $chat_id,
+            'parse_mode' => 'Markdown',
```
Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Not that it makes any difference, but no need for `parse_mode` here.

File: src/Commands/AdminCommands/CleanupCommand.php (hunk 1 above)
Comment (jacklul) [inline src/Commands/AdminCommands/CleanupCommand.php]: Because... `getFrom()->getId()` would return id of the user who called this command, so the bot would reply to the private chat with that user.

File: src/Commands/AdminCommands/CleanupCommand.php (hunk 2 above)
Comment (jacklul) [inline src/Commands/AdminCommands/CleanupCommand.php]: Now it does!

File: src/Commands/AdminCommands/CleanupCommand.php (hunk 1 above)
Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Yes, I know that, which is why I thought you put that there in the first place :smile:
Because with `getChat`, it posts any error or the final summary visible for anyone in a chat to read.
IMHO, maintenance stuff should be managed privately.

File: src/Commands/AdminCommands/CleanupCommand.php (hunk 2 above)
Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Haha, you crack me up... 😉

File: src/Commands/AdminCommands/CleanupCommand.php (hunk 1 above)
Comment (jacklul) [inline src/Commands/AdminCommands/CleanupCommand.php]: Very rare case would be an admin who didnt message his bot in private chat yet.
All admins commands should get some kind of change like this, maybe we could decide on something and do it in this PR?

Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Agreed, it's a rare case, but it can happen by mistake. Using `getFrom()` is a fail-safe for that, as the Chat ID would be the same as the From ID anyway in a private chat, so it doesn't make a difference there.

Haven't tested this yet, but can a bot delete a message from a user when it's admin in a group/channel? Then, we could even go as far as to offer the option of deleting a "by mistake" command execution.

What do you think?

Comment (jacklul) [inline src/Commands/AdminCommands/CleanupCommand.php]: Yes, bot can delete user messages when he has the permissions to do so.

That would be a nice solution!

Comment (noplanman) [inline src/Commands/AdminCommands/CleanupCommand.php]: Ok, then I think that's what we should go for 👍
Will play around with this when I have a moment.
